client/src/holdings/HoldingsList.tsx

```diff
@@ -178,7 +179,11 @@ export function HoldingsList({user, fxRates}: IProps) {
   }
 
   if (!isDataLoaded) {
-    return <div className='loading'>Loading...</div>;
+    return (
+      <div className='loading'>
+        <CircularProgress />
+      </div>
+    );
```

Comment:
Nice, can you do the same for the one in App.tsx? Also it would be nice if it could be put in the center of the page, atm it's in the top left (but this last bit isn't essential)